src/DssVest.sol

```diff
@@ -229,7 +229,7 @@ abstract contract DssVest is ERC2771Context, Initializable {
         @param _mgr An optional manager for the contract. Can yank if vesting ends prematurely.
         @return id  The id of the vesting contract
     */
-    function create(address _usr, uint256 _tot, uint256 _bgn, uint256 _tau, uint256 _eta, address _mgr) external lock returns (uint256 id) {
+    function create(address _usr, uint256 _tot, uint256 _bgn, uint256 _tau, uint256 _eta, address _mgr) external lock auth returns (uint256 id) {
```

Comment:
Why is this function external?

Isn't it enough for `commit` and `claim` to be public?

Reply:
It would be enough if we wanted to only allow the `commit` and `claim` approach to work. But with this additional function (that only calls the internal `_create`), the traditional `create` still works.
This way, we stay closer to the original implementation and need less transactions to create a public vesting plan.

Reply:
But in the original version it is internal.
There the `createAward` was used, right?

Since the function doesn't exist anymore, we should remove it in the comments

Reply:
The [original function is called](https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L185) `create`, too.

When you sketched blind vesting, you chose `createAward` and I missed the references during renaming. Updated the comments as you suggested. Thanks for checking so thoroughly! 
[e63ad62](https://github.com/corpus-io/dss-vest/pull/37/commits/e63ad626e374752e5aa3d9fad5c9e1feaeffed6d)